### ISSUE TITLE
SOLVED: Getting Simplified Chinese transcriptions instead of Traditional Chinese

### ISSUE THREAD

**Issue by Charles-Leo** [octimot/StoryToolkitAI#55]
I tried the mac version and the windows version, but the windows version does not support Simplified Chinese, so the next version can provide an option. In addition, it is hoped that the execution log window will be integrated into the main interface of the software.

**Comment by octimot** [comment]
> I tried the mac version and the windows version, but the windows version does not support Simplified Chinese, so the next version can provide an option.

Thanks for the feedback! Are you referring to the user interface or the transcription function?

> In addition, it is hoped that the execution log window will be integrated into the main interface of the software.

Do you mean that you'd like to see the output of the terminal in the main interface, or? Also, could you please open a separate issue for this request? Thanks!

**Comment by Charles-Leo** [comment]
Thanks for your reply, I was referring to the transcription function. Also, if the user interface could also have an option for Simplified Chinese, that would be appreciated.

**Comment by octimot** [comment]
We're not planning to develop the UI in other languages except English for now, but the transcriptions should work.

But, just to clarify:

1. Do you see Simplified Chinese text in the transcription window, once the transcription process is finished?
2. Can you import the SRT files into Resolve (or other software), once the transcription process is finished, if yes - do the characters show up correctly?
3. Which transcription model are you using?

**Comment by Charles-Leo** [comment]
I use the mac version to transcribe files and can display Simplified Chinese normally, but the Windows version displays Traditional Chinese, and other functions are normal. So, can you provide an option for Simplified Chinese and Traditional Chinese? I have tried Tiny, small and medium and the results are the same.

**Comment by octimot** [comment]
Sorry, I don't understand. Could you please reply on the above questions - referring to the Windows version that you have problems with - so that we have a better picture of what's going on? Thanks!

**Comment by Charles-Leo** [comment]
When I select the transcription language as Chinese, the model selection is medium, the transcription result of the mac version is Simplified Chinese, and the transcription result of the Windows version is Traditional Chinese.

**Comment by octimot** [comment]
The different behaviour is possible because you're most likely using very different CPU/GPU configurations between the Mac and Windows systems.

First, I suggest using either the medium or the large model for non-english audio. 

Second, in the Transcription Settings window, try to give it an Initial Prompt that is in Simplified Chinese, but use the same dialogue structure as you have it by default in English so that it understands that there is a dialogue happening in the audio. For eg.:

```
- 以下是普通话句子
- 好的
```

Let me know if this works!

**Comment by Charles-Leo** [comment]
OK, Thanks for your suggestion，I`ll try it.

**Comment by Charles-Leo** [comment]
Add the following Initial Prompt，
- 以下是普通话句子
- 好的

 It`s get work. Thansk you!

**Comment by octimot** [comment]
Cool! I'm glad it does!

On the next update, you'll also have access to Whisper large-v2 model which may work even better in some cases.
